esmf_regrid/esmf_regridder.py

```diff
@@ -185,7 +186,7 @@ def _array_to_matrix(self, array):
         We then take the transpose so that matrix multiplication happens over
         the appropriate axes.
         """
-        return array.reshape(-1, (self.size()), order="F").T
+        return array.reshape(-1, (self.size), order="F").T
```

Comment:
I like the property :smile: but I was talking about the parentheses that are now left around `self.size`, ie
```suggestion
        return array.reshape(-1, self.size, order="F").T
```